temporal-test-server/src/main/java/io/temporal/internal/testservice/TestWorkflowMutableStateImpl.java

```diff
@@ -2218,6 +2234,32 @@ public void completeNexusOperation(NexusOperationRef ref, Payload result) {
         });
   }
 
+  @Override
+  public void completeAsyncNexusOperation(
+      NexusOperationRef ref,
+      Payload result,
+      String operationID,
+      io.temporal.api.nexus.v1.Link startLink) {
+    update(
+        ctx -> {
+          StateMachine<NexusOperationData> operation =
+              getPendingNexusOperation(ref.getScheduledEventId());
+          if (operation.getState() == State.INITIATED) {
```

Comment:
To save me from reading, does this cover when the activity is backing off and scheduled?

Reply:
Yes, it goes back to INITIATED when backing off.